### PR TITLE
Fix add-copyright year function

### PR DESCRIPTION
**Pull request by superdyoll**
* Fixes bug raised in [#414](https://github.com/ament/ament_lint/issues/414)
Signed-off-by: Lloyd Pearson <superdyoll@gmail.com>

### CODE REVIEW

**Comment by superdyoll** [comment]
@clalancette is there anything you'd like me to change, or are you happy to merge this?

File: ament_copyright/ament_copyright/main.py [hunk 1]
```diff
@@ -287,39 +287,40 @@ def add_copyright_year(file_descriptors, new_years, verbose):
...
+                file_descriptor.content[global_years_span[1]:]
+
+            # output beginning of file for debugging
+            # index = global_years_span[0]
+            # for _ in range(3):
+            #     index = get_index_of_next_line(content, index)
+            # print('<<<')
+            # print(content[:index - 1])
+            # print('>>>')
+
```
Comment (clalancette) [inline ament_copyright/ament_copyright/main.py:321]: I know you didn't add this, but let's just remove this debugging code.

**Review by clalancette — CHANGES_REQUESTED** [review]
In short, this looks great.  I've left one comment to cleanup; once we do that, I'm happy to approve and run CI on it.

File: ament_copyright/ament_copyright/main.py (hunk 1 above)
Comment (superdyoll) [reply ament_copyright/ament_copyright/main.py]: Done :)

**Review by clalancette — APPROVED** [review]

**Pull request merged**